scripts/zones/Batallia_Downs/mobs/Lumber_Jack.lua

```diff
@@ -8,8 +8,10 @@ require("scripts/globals/mobs")
 local entity = {}
 
 entity.onMobInitialize = function(mob)
+    mob:setMod(xi.mod.UFASTCAST, 100)
     mob:setMobMod(xi.mobMod.ADD_EFFECT, 1)
     mob:setMobMod(xi.mobMod.IDLE_DESPAWN, 600)
+    mob:setMod(xi.mod.DOUBLE_ATTACK, 20)
```

Comment:
Despite what clopedia says 
![image](https://user-images.githubusercontent.com/6871475/178092604-0ec3096b-248f-4556-81b7-dc1eda400b67.png)
he probably gets this from a war subjob

Reply:
if his nukes are garbage with a war sub, leave his sub alone and keep the mod its conjecture either way, just looking for path of least resistance in mob building. I'll trust you judgement on this.

Reply:
His damage if unresisted still seems fine but taking off the BLM sub his magic accuracy is real bad. I think I would prefer to keep the BLM sub and just give him the double attack mod.

Reply:
ok. thanks for checking.